From 46c7bdb6ff065809273bc83d62d3ae9074750452 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Wed, 14 Dec 2016 16:21:51 +0800
Subject: [PATCH 1376/5242] MLK-13607-8 video: mipi-panel: hx8363: change DE
 polarity to active high

commit  599c8dfea08987a7a39b5966fd01d75ca7a91879 from
https://source.codeaurora.org/external/imx/linux-imx.git

The NorthWest Logic MIPI DSI only support active low data enable
signal on imx7ulp-evk board.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit 33781fd4cf41e2b0f7dfc60b5527755a57aa6aa3)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/mxcfb_hx8363_wvga.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/fbdev/mxc/mxcfb_hx8363_wvga.c b/drivers/video/fbdev/mxc/mxcfb_hx8363_wvga.c
index be97e81..d6052bc 100644
--- a/drivers/video/fbdev/mxc/mxcfb_hx8363_wvga.c
+++ b/drivers/video/fbdev/mxc/mxcfb_hx8363_wvga.c
@@ -134,7 +134,7 @@ static void parse_variadic(int n, u8 *buf, ...)
 	 40, 60,
 	 3, 3,
 	 8, 4,
-	 FB_SYNC_OE_LOW_ACT,
+	 0x0,
 	 FB_VMODE_NONINTERLACED,
 	 0,
 	},
-- 
1.7.9.5

